apps/api/src/app/topics/e2e/add-subscribers.e2e.ts

```diff
@@ -0,0 +1,116 @@
+    const subscribers = [subscriber._id];
+
+    const response = await session.testAgent.post(addSubscribersUrl).send({ subscribers });
+
+    expect(response.statusCode).to.eql(204);
+    expect(response.body).to.be.empty;
+
+    const getResponse = await session.testAgent.get(getTopicUrl);
+    expect(getResponse.statusCode).to.eql(200);
+
+    const getResponseTopic = getResponse.body.data;
+
+    expect(getResponseTopic._id).to.eql(topicId);
+    expect(getResponseTopic._userId).to.eql(session.user._id);
+    expect(getResponseTopic._environmentId).to.eql(session.environment._id);
+    expect(getResponseTopic._organizationId).to.eql(session.organization._id);
+    expect(getResponseTopic.key).to.eql(topicKey);
+    expect(getResponseTopic.name).to.eql(topicName);
+    expect(getResponseTopic.subscribers).to.eql([subscriber._id]);
+  });
+
+  it('should not duplicate subscribers if adding a subscriber already added to topic', async () => {
+    const preconditionResponse = await session.testAgent.get(getTopicUrl);
+    expect(preconditionResponse.statusCode).to.eql(200);
+    expect(preconditionResponse.body.data.subscribers).to.eql([subscriber._id]);
+
+    const subscribers = [subscriber._id];
+
+    const response = await session.testAgent.post(addSubscribersUrl).send({ subscribers });
+
+    expect(response.statusCode).to.eql(204);
+
+    const getResponse = await session.testAgent.get(getTopicUrl);
+    expect(getResponse.statusCode).to.eql(200);
+
+    const getResponseTopic = getResponse.body.data;
+
+    expect(getResponseTopic._id).to.eql(topicId);
+    expect(getResponseTopic._userId).to.eql(session.user._id);
+    expect(getResponseTopic._environmentId).to.eql(session.environment._id);
+    expect(getResponseTopic._organizationId).to.eql(session.organization._id);
+    expect(getResponseTopic.key).to.eql(topicKey);
+    expect(getResponseTopic.name).to.eql(topicName);
+    expect(getResponseTopic.subscribers).to.eql([subscriber._id]);
+  });
+
+  it('should add multiple subscribers to topic', async () => {
+    const subscribers = [secondSubscriber._id, thirdSubscriber._id];
+
+    const response = await session.testAgent.post(addSubscribersUrl).send({ subscribers });
+
+    expect(response.statusCode).to.eql(204);
+
+    const getResponse = await session.testAgent.get(getTopicUrl);
+    expect(getResponse.statusCode).to.eql(200);
+
+    const getResponseTopic = getResponse.body.data;
+
+    expect(getResponseTopic._id).to.eql(topicId);
+    expect(getResponseTopic._userId).to.eql(session.user._id);
+    expect(getResponseTopic._environmentId).to.eql(session.environment._id);
+    expect(getResponseTopic._organizationId).to.eql(session.organization._id);
+    expect(getResponseTopic.key).to.eql(topicKey);
+    expect(getResponseTopic.name).to.eql(topicName);
+    expect(getResponseTopic.subscribers).to.eql([subscriber._id, secondSubscriber._id, thirdSubscriber._id]);
+  });
+});
```

Comment:
missing the test that will add subscribers to the existing array of subscribers

Reply:
Is this one. Our test suites don't reset the database and they are executed sequentially.

Reply:
you can just create a new topic with the subscribers and then add other subscribers to it

Reply:
my suggestion is to add a brand new test ;) I've selected line 116 to leave the message... 
in the test what you can do is to create a new topic then add subscribers and then again add new subscribers, at the end check if all are included